otter/models/cass.py

```diff
@@ -444,11 +448,27 @@ def _jsonloads_data(raw_data):
     return data
 
 
+def _group_status(status, deleting):
+    if deleting:
+        return ScalingGroupStatus.DELETING
+    else:
+        # TODO: #1304
```

Comment:
Actually it applies to both the `None` and the `'DISABLED'` since the ticket is about migrating both.